webpack.config.js

```diff
@@ -2,14 +2,13 @@
 
 const merge = require('webpack-merge')
 const {
-  production,
   target,
   hotReload,
   analyze
 } = require('./config/webpack.vars')
 
 module.exports = (env = {}) => {
-  env.target = env.target || target
+  env.target = env.target || 'browser' 
```

Comment:
Maybe we should have a default env and use it like this?

```js
const defaultEnv = {
  target: 'browser'
}

module.exports = (env = defaultEnv) => {
...
}
```

Reply:
Yes, though I would not use the default argument syntax since otherwise, inside the function you are changing the defaultEnv object and this is not something you want.

Done in https://github.com/cozy/cozy-banks/pull/1120/commits/4bef509bb4f421b9fd71be810255afcf19453c71